lib/constants.php

```diff
@@ -3,3 +3,18 @@
 namespace Amp\Process;
 
 const BIN_DIR = __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'bin';
+const IS_WINDOWS = (PHP_OS & "\xDF\xDF\xDF") === 'WIN';
```

Comment:
wtf? Okay, I see. `~"\x20\x20\x20"` would've been probably a little more obvious.

I want pure functions in PHP which can be compiletime evaluated...